RELEASE_NOTES.md

```diff
@@ -14,6 +14,52 @@ report it to our [issue tracker].
 
 # Release Notes
 
+## 2024-08-07 - Kanidm 1.3.0
+
+This is the latest stable release of the Kanidm Identity Management project. Every release is
+the combined effort of our community and we appreciate their invaluable contributions, comments,
+questions, feedback and support.
+
+You should review our
+[support documentation](https://github.com/kanidm/kanidm/blob/master/book/src/support.md)
+as this may have important effects on your distribution or upgrades in future.
+
+Before upgrading you should review [our upgrade documentation](https://kanidm.github.io/kanidm/stable/server_updates.html#general-update-notes)
+
```

Comment:
We should stick to a single linking strategy - one links to the book source in master, the other links to the compiled book in stable? 😄

Reply:
Oopppp,s good point.